docs/example-data.json

```diff
@@ -194,6 +194,8 @@
 
         "conference_show": false,
         "conference_auto_connect": false,
+        "conference_los_restriction": false,
+        "stream_url": "",
```

Comment:
prefix `conference_`?

Reply:
Strictly, this isn't a setting for the conference, that's why in OS3 the prefix isn't present either. But I can add it if you want

Reply:
Done